doc/080_examples.rst

```diff
@@ -202,12 +202,13 @@ configuration of restic will be placed into environment variables. This will
 include sensitive information, such as your AWS secret and repository password.
 Therefore, make sure the next commands **do not** end up in your shell's
 history file. Adjust the contents of the environment variables to fit your
-bucket's name and your user's API credentials.
+bucket's name, region, and your user's API credentials.
 
 .. code-block:: console
 
    $ unset HISTFILE
-   $ export RESTIC_REPOSITORY="s3:https://s3.amazonaws.com/restic-demo"
+   $ export AWS_REGION="eu-west-1"
+   $ export RESTIC_REPOSITORY="s3:https://s3-${AWS_REGION}.amazonaws.com/restic-demo"
```

Comment:
I think we should stay consistent with https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html#amazon-s3 here and just export `AWS_DEFAULT_REGION`.

Or at least use the punctuation style s3 hostname, that is `s3.${AWS_REGION}.amazonaws.com` and maybe update 030_preparing_a_new_repo.rst to always recommend specifying a region.